EnglishTranslation-master/ETC.tsv

```diff
@@ -16320,7 +16320,7 @@ ETC_20151022_016321	Lighting up the bonfire
 ETC_20151022_016322	You've already used it
 ETC_20151022_016323	Finally, Orsha!
 ETC_20151022_016324	You've helped the settlers by lighting the fire!
-ETC_20151022_016325	You've obtained the usable objects!
+ETC_20151022_016325	Collected Useful Objects!
```

Comment:
Does the message appear everytime you obtain 1 since you need 5? If so: "You've obtained a Useful Object".

I'll change the item name to "Useful Object"